OpenWeather/Persistence/Database/Implementation/SQLiteDatabaseService.swift

```diff
@@ -0,0 +1,336 @@
+//
+//  OpenWeather template generated by OpenBytes on 30/04/2023.
+//
+// Created by Ahmed Shendy.
+//  SQLiteDatabaseService.swift
+//
+
+import GRDB
+import os.log
+import Combine
+import Foundation
+
+struct SQLiteDatabaseService: DatabaseService {
+    private let dbWriter: any DatabaseWriter
```

Comment:
Can we have another value for a DatabaseReader? Does that exist in GRDB?

Also do we need the `any` keyword here?